code/ecdsap256/Hacl.P256.fsti

```diff
@@ -19,10 +19,30 @@ open Spec.ECDSAP256.Definition
 open Hacl.Impl.P256.Compression
 open Spec.P256.MontgomeryMultiplication
 
+[@@ CPrologue "
+/*******************************************************************************
 
-[@ (Comment " Input: result buffer: uint8[64], \n m buffer: uint8 [mLen], \n priv(ate)Key: uint8[32], \n k (nonce): uint32[32]. 
+ECDSA and ECDH functions over the P-256 NIST curve.
+
+This module implements signing and verification, key validation, conversions
+between various point representations, and ECDH key agreement.
+
+*******************************************************************************/
+
+/**************/
+/* Signatures */
+/**************/
+
+/*
+  The standard strongly encourages the user to hash the input before signing
```

Comment:
I am not sure about "strongly encourages". Usually, the hash of the message is signed, true, but I think "strongly encouraged" gives an impression of an attack if it is not the hash

Reply:
Can you go ahead and suggest an alternate phrasing? Don't hesitate to commit on the branch.

Reply:
I've reworded to keep closer to the standard.